editor.planx.uk/src/pages/Team/components/ShowingServicesHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import { FlowSummary } from "pages/FlowEditor/lib/store/editor";
+import React, { useEffect, useState } from "react";
+
+export const ShowingServicesHeader = ({
+  matchedFlows,
+}: {
+  matchedFlows: FlowSummary[] | null;
+}) => {
+  const [numberOfFlows, setNumberOfFlows] = useState(matchedFlows?.length || 0);
+
+  useEffect(() => {
+    setNumberOfFlows(matchedFlows?.length || 0);
+  }, [matchedFlows]);
```

Comment:
Is this internal state actually required - can we just pass a count into the component?

```ts
<ShowingServicesHeader flowCount={matchingFlows.length} />
```